easybuild/easyconfigs/b/BEEF/BEEF-0.1.1-r16-iomkl-4.6.13.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Benjamin P. Roberts
+# New Zealand eScience Infrastructure
+# The University of Auckland, Auckland, New Zealand 
+
+easyblock = 'ConfigureMake'
+
+name = 'BEEF'
+version = '1.0-r16'
+
+homepage = 'http://suncat.stanford.edu/facility/software/functional'
+description = """BEEF is a library-based implementation of the Bayesian
+Error Estimation Functional, suitable for linking against by Fortran-
+or C-based DFT codes. A description of BEEF can be found at
+http://dx.doi.org/10.1103/PhysRevB.85.235149."""
+
+toolchain = {'name': 'iomkl', 'version': '4.6.13'}
+toolchainopts = {'openmp': False, 'usempi': False}
+
+# There is no source_url as this version is built from a tarred SVN checkout.
```

Comment:
maybe include the SVN repo URL?
and also the`svn`  command one can use to check out this exact version?

you'll need to fix the easyconfig filename here too, the versions don't match up